cashctrl_api/client.py

```diff
@@ -212,39 +212,71 @@ def flatten_nodes(nodes, parent_path=''):
 
         data = self.get(f"{resource}/category/tree.json")['data']
         df = pd.DataFrame(flatten_nodes(data.copy()))
-        df = enforce_dtypes(df, CATEGORY_COLUMNS)
+
+        # Add number column for account categories
+        if resource == 'account':
+            accounts = pd.DataFrame(self.get("account/list.json")['data'])
+            accounts = accounts.groupby('categoryId', as_index=False).agg({'number': 'min'})
+            df.drop(columns=['number'], inplace=True)
+            df = pd.merge(df, accounts, how='left', left_on='id', right_on='categoryId')
```

Comment:
I do not understand this close block. The category number (not to be confused with an account number) should already be part of `df`. I believe there is nothing to do.